cl/citations/management/commands/import_citations_csv.py

```diff
@@ -0,0 +1,128 @@
+"""
+Import citations from csv file
+
+The csv file must have the following structure, the file should not have a header row:
+
+"2155423", "2003 WL 22508842"
+"7903720","520 A.2d 234"
+"7903715","520 A.2d 233"
+
+How to run the command:
+manage.py import_citations_csv --csv /opt/courtlistener/cl/assets/media/wl_citations_1.csv
+
+Note: If --limit is greater than --end-row, end row will be ignored
+
+"""
+
+import os.path
+
+import numpy as np
+import pandas as pd
+from django.core.management import BaseCommand
+from pandas import DataFrame
+from pandas.io.parsers import TextFileReader
+
+from cl.corpus_importer.utils import add_citations_to_cluster
+from cl.lib.command_utils import logger
+from cl.search.models import OpinionCluster
+
+
+def load_citations_file(options: dict) -> DataFrame | TextFileReader:
+    """Load csv file from absolute path
+
+    :param options: options passed to command
+    :return: loaded data
+    """
+
+    start_row = None
+    end_row = None
+
+    if options["start_row"] and options["end_row"]:
+        start_row = options["start_row"] if options["start_row"] > 1 else 0
+        end_row = options["end_row"] - options["start_row"] + 1  # inclusive
+
+    if options["limit"]:
+        end_row = options["limit"]
+
+    data = pd.read_csv(
+        options["csv"],
+        names=["cluster_id", "citation_to_add"],
+        delimiter=",",
+        skiprows=start_row,
+        nrows=end_row,
+    )
```

Comment:
I think this start_row end_row assigning of variable doesnt make any sense to me.  Doesnt this mean that we are always putting None into the skip rows and nrows variable?